.github/workflows/r-conda.yml

```diff
@@ -51,4 +51,5 @@ jobs:
       - name: Test coverage
         run: |
           conda activate recetox-xmsannotator-dev
-          Rscript -e "covr::codecov('xmsannotator', quiet = FALSE)"
\ No newline at end of file
+          options(covr.gcov="")
+          Rscript -e "covr::codecov('xmsannotator', quiet = FALSE)"
```

Comment:
You can do it like this 
`Rscript -e "options(covr.gcov=''); covr::codecov('xmsannotator', quiet = FALSE)"`

Reply:
Oh right, it's an R code. Thanks!